numpy/core/_add_newdocs.py

```diff
@@ -3939,17 +3939,22 @@
 
     Constructs Python bytes showing a copy of the raw contents of
     data memory. The bytes object can be produced in either 'C' or 'Fortran',
```

Comment:
Also, I think the description of the order kwargs here is redundant (or will be, once the `order` parameter description from `copy` is included). Most of specific wording here could be replaced with something like:

> The bytes object is produced in C-order by default. This behavior is controlled by the ``order`` parameter.

Reply:
Got it